src/citrine/gemtables/columns.py

```diff
@@ -396,13 +396,14 @@ def __init__(self, *, data_source: str, format: ChemicalDisplayFormat):
 
 
 class ConcatColumn(Serializable['ConcatColumn'], Column):
-    """[ALPHA] Column that concatenates .
+    """[ALPHA] Column that concatenates multiple values.
 
     Parameters
     ----------
     data_source: str
         name of the variable to use when populating the column
-
+    subcolumn: Column
+        a column of the type of the individual values to be concatenated
```

Comment:
I know this isn't the point of this review, but would this work for ambiguous values? That would be awesome

Reply:
I don't think so

Reply:
@jspeerless All this column does is concatenate a column definition mapped across a List or Set.  If we were to support what you are asking for above, this might be the column people use.  However, we'd need a new variable definition to accomplish what you ask.